app/views/questions/_answer.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="answers-container" style="margin-bottom:30px;">
+  <div id="a<%= answer.id %>" style="border: 1px solid #337ab7; border-radius:5px;">
```

Comment:
You can see there are some different classes and variables involved in the header so it is not same as the comments header.